retail/solutions/frontline-worker/README.md

```diff
@@ -0,0 +1,145 @@
+# Real-time store communications and collaboration
```

Comment:
@andrehass , we need to change the structure of this doc. Please use this as a guide to restructure this doc? 
https://github.com/microsoft/industry/tree/main/retail/solutions/flexibleFulfillment

This will make it consistent with other guidances we have just created for other industries (FSI, health etc.).

Reply:
Also, for the architecture for individual solutions, we are using the architecture which was created in the Visio (created by Sanem) as foundations or starting point for various retail solutions. We want to represent the architecture in a consistent fashion.  I have shared Visio via Teams chat. You can re-create architecture for your solution there.

Refer to the architecture diagrams for the following solutions: ttps://github.com/microsoft/industry/tree/main/retail/solutions/flexibleFulfillment
https://github.com/microsoft/industry/tree/main/retail/solutions/realtimePersonalization